contracts/src/v0.8/dev/ocr2/OCR2BaseUpgradeable.sol

```diff
@@ -0,0 +1,397 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../ConfirmedOwnerUpgradeable.sol";
+import "./OCR2Abstract.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
```

Comment:
We don't technically need this import as it's implicitly imported via `ConfirmedOwnerUpgradeable`. However, as per our guidelines, let's be explicit about every import in every file so we can reason about inheritance more easily:

```
import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
```

We have [Chainlink Solhint](https://github.com/smartcontractkit/chainlink-solhint-rules) rules to enforce this, can we apply these here.